test/string/memmove.c

```diff
@@ -14,18 +15,92 @@
 #include <cmocka.h>
 // clang-format on
 
-static void memmove_test(void** state)
+struct test_memmove_data {
+	char template[64];
+	char buf[64];
+};
+
+static void fill_buffer(char* buf, size_t size, char start_value, char end_value)
+{
+	size_t i;
+	for (i=0; i< size; ++i)
```

Comment:
Please add braces around the body of the `for`

Reply:
Thanks, I was just on my default Linux code style and forgot to check what was happening here :).